common/h/registers/x86_regs.h

```diff
@@ -238,6 +249,7 @@ namespace Dyninst { namespace x86 {
   DEF_REGISTER(      xmm5,   0x05 |  XMMS |   XMM | Arch_x86, "x86");
   DEF_REGISTER(      xmm6,   0x06 |  XMMS |   XMM | Arch_x86, "x86");
   DEF_REGISTER(      xmm7,   0x07 |  XMMS |   XMM | Arch_x86, "x86");
+  DEF_REGISTER(     mxcsr,   0x08 |  FULL | FPCTL | Arch_x86, "x86");
```

Comment:
mxcsr is 32-bits so `FULL` should be `D_REG`

Reply:
This is for x86_i386, so FULL is 32-bit. It is D_REG in x86_64_regs.h.